modules.d/01systemd-modules-load/module-setup.sh

```diff
@@ -27,20 +27,19 @@ depends() {
 install() {
 
     # Create systemd-modules-load related directories.
-    inst_dir $modulesload
-    inst_dir $modulesloadconfdir
+    inst_dir "$modulesload"
+    inst_dir "$modulesloadconfdir"
```

Comment:
These global variables don't exist.

Reply:
Any idea where this comes from then? Otherwise let's :fire: it.

Reply:
@danimo it's a needed global variable which is missing since that global variable ended up as part of another sd-module work. Work which I put on ice until @haraldh system wide fixes, which touch a large chunk of the code base settles down. ( we are just signing ourselves up for merge conflicts otherwise ) which hopefully is done now for the time being.
@haraldh has created issues for this and assigned it to me to clean up ( issues #1262 #1263 ) and that work kinda depends on @haralds draft PR #1110 being finished as in it's better that fix it there after we merged that work.

Reply:
@johannbg So we keep this as-is for now?

Reply:
@danimo if you mean "keep this as-is" as being "approve @haraldh PR and merge as is" then the answer is yes.

Reply:
Hmm, it was not working before and not working after this patch... So, I don't know.

Reply:
Create an issue and I'll approve. Let's not change too much at once in a single commit.

Reply:
@danimo @haraldh already created an issue for missing global variables in SD modules and assigned it to me.